Makefile

```
@@ -0,0 +1,21 @@
+CPYTHON_PATH               = ../cpython
+PACKAGE_ABS_PATH = $(shell pwd)/$(shell find dist/python-docs-theme-*.tar.gz)
+
+
+.PHONY: all
+all: install
+	cd $(CPYTHON_PATH)/Doc && \
+		make html
+
+
+.PHONY: install
+install:
+	python3 -m build
+	cd $(CPYTHON_PATH)/Doc && \
+		./venv/bin/pip install $(PACKAGE_ABS_PATH)
```

And we need a `make venv` in here, as the CPython repo might not have a venv in its `Doc` dir yet (also align ampersands for readability):
```suggestion
	cd $(CPYTHON_PATH)/Doc \
		&& make venv \
		&& ./venv/bin/pip install $(PACKAGE_ABS_PATH)
```